kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -723,6 +753,31 @@ def test_delete_submission_as_owner(self):
 
         self.assertEqual(response.data['count'], len(self.submissions) - 1)
 
+    def test_audit_log_on_delete(self):
+        """
+        Validate that the submission id is logged in AuditLog table when it is
+        deleted.
+        """
+        submission = self.submissions_submitted_by_someuser[0]
+        (
+            app_label,
+            model_name,
+        ) = self.asset.deployment.submission_model.get_app_label_and_model_name()
+        audit_log_count = AuditLog.objects.filter(
+            user=self.someuser, app_label=app_label, model_name=model_name
+        ).count()
+        # No submissions have been deleted yet
+        assert audit_log_count == 0
+        # Delete one submission
+        self.test_delete_submission_as_owner()
+
+        # All submissions have been deleted and should be logged
+        deleted_submission_ids = AuditLog.objects.values_list(
+            'pk', flat=True
+        ).filter(user=self.someuser, app_label=app_label, model_name=model_name)
+        assert len(deleted_submission_ids) != audit_log_count
```

Comment:
As mentioned earlier, just for readability, it might be nice not to use the name `audit_log_count` here